libs/components/indicators/src/lib/modules/tokens/fixtures/tokens.component.fixture.ts

```diff
@@ -44,7 +45,19 @@ export class SkyTokensTestComponent implements OnDestroy {
 
   public onFocusIndexUnderRange(): void {}
 
-  public onTokenSelected(): void {}
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I added a couple small eslint things just to clean up editors - these weren't true issues or things we need to clean up long term.